src/frontend/src/common/components/library/data_subview/components/DownloadModal/index.tsx

```diff
@@ -289,6 +290,14 @@ const DownloadModal = ({
                   </DownloadTypeInfo>
                 </Alert>
               )}
+            {checkedSampleIds.length - failedSampleIds.length > 999 &&
```

Comment:
I removed failed samples from this count since we wouldn't be putting them into the file anyway.  Is this correct?

Reply:
That seems reasonable to me? If you verified that failed samples don't get put into the download -- I think you're right, just haven't looked myself -- than this seems good to me.

Reply:
@phoenixAja I'm going to go ahead and merge this, but lmk if it needs alteration.